RPI/Top/instances.fpp

```diff
@@ -313,18 +311,13 @@ module RPI {
   instance rateGroupDriverComp: Svc.RateGroupDriver base id 1700 \
   {
 
-    phase Fpp.ToCpp.Phases.configObjects """
+    phase Fpp.ToCpp.Phases.configComponents """
```

Comment:
Make sure it's OK to declare this array inside a function (array is copied before use).